lib/ts/recipe/session/types.ts

```diff
@@ -75,7 +75,7 @@ export interface ErrorHandlers {
     onInvalidClaim?: InvalidClaimErrorHandlerMiddleware;
 }
 
-export type TokenType = "access" | "refresh" | "idRefresh";
+export type TokenType = "access" | "refresh";
```

Comment:
there is a type below called `CreateOrRefreshAPIResponse` which has an IdrefreshToken section - remove that?